specification/workloads/resource-manager/Microsoft.Workloads/monitors/readme.go.md

```diff
@@ -0,0 +1,34 @@
+## Go
```

Comment:
Its not in sync with SVI readme.go.md file. Please check this file [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/readme.go.md?plain=1](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/readme.go.md?plain=1)